src/ansys/aedt/toolkits/antenna/backend/common/api_generic.py

```diff
@@ -445,14 +447,16 @@ def connect_aedt(self):
         return True
 
     def connect_design(self, app_name=None):
-        """Connect to an application design.
-        If a design exists, it takes the active project and design, if not,
-        it creates a new design of the specified type. If no application specified, the default is ``"Hfss"``.
+        """Connect to a design in an AEDT app.
+
+        If a design exists, this method takes the active project and design. If a design does
+        not exist, this method creates a design of the specified type.
 
         Parameters
         ----------
         app_name : str, optional
-            Aedt application name. The default is connecting to active design. Application available are:
+            AEDT app name.  If no app is specified, the default is ``"Hfss"``.
+            Options are:
```

Comment:
@Samuelopez-ansys It looks like the following list of options should be what the user types (and be alphabetized). I think it should look like this, but I want you to decide. (And remember that for this list to render correctly in the doc, you must have a blank line before the first list tiem and after the last list item.)

- ``"Circuit"``
- ``"Emit"``
- ``"Hfss"``
- ``"Hfss3dLayout"``
- ``"Icepak"``
- ``"Maxwell2d"``
- ``"Maxwell3d"``
- ``"MaxwellCircuit"``
- ``"Mechanical"``
- ``"Q2d"``
- ``"Q3d"``
- ``"Rmxprt"``
- ``"Simplorer"``